### DISCUSSION TITLE
[Feature]: Any plans for supporting lit ?

### DISCUSSION THREAD

**Discussion by MaxtuneLee** [web-infra-dev/rspack#6480] [Ideas]
Currently rspack does not support lit yet. While vite supports lit, and it is really useful to start a lit project for application or package.
Lit has a guide to pack production code: https://lit.dev/docs/tools/production/

**Comment by xc2** [comment]
https://github.com/rspack-contrib/rspack-examples/tree/main/rsbuild/lit

There's an example of building lit project with rsbuild.

**Reply by MaxtuneLee** [reply]
That's so great to hear, thanks!
